### DISCUSSION TITLE
Multiple hosts / master & replicas

### DISCUSSION THREAD

**Discussion by KirillDyachkovskiy** [timgit/pg-boss#411] [Q&A] [answered]
Dear esteemed developer of PGBoss library,

I hope this message finds you well. I have been exploring various options for managing databases and came across the PGBoss library. I am highly interested in utilizing it for my project, but I would like to inquire whether the library supports multiple hosts and allows for master and replica separation. This feature would be crucial for ensuring high availability and scalability in my application.

Thank you in advance for your response

**Comment by timgit** [comment, marked as answer]
This should be possible to set up, but it seems to be blocked by #387 before it can be implemented
